package.json

```diff
@@ -21,6 +21,7 @@
     "lint:sass": "stylelint \"**/*.{css,scss}\" && prettier --check \"**/*.{css,scss}\"",
     "lint:yml": "prettier --check \"**/*.{yaml,yml}\"",
     "lint": "yarn lint:js && yarn lint:json && yarn lint:sass && yarn lint:yml",
+    "postinstall": "cd streaming && yarn",
```

Comment:
@Gargron if you want, we can keep this, or we can get rid of it — the benefit of keeping it until the next major is that we wouldn't break how people upgrade their servers, they can still do a `yarn install` and get everything they need installed, despite under the hood the package.json files have been split.

I see this as a stop-gap until we can probably define the boundaries/protocols between rails and streaming, and ultimately rewrite streaming in Go, which would be done as a major-release change.